### ISSUE TITLE
Proposal to improve the transfer learning tutorial

### ISSUE THREAD

**Issue by youkaichao** [pytorch/tutorials#1195]
Dear Pytorch tutorial developers,

I'm a researcher in transfer learning ([my research profile](https://scholar.google.com/citations?user=Ke4w3KkAAAAJ&hl=en)) and happen to read the transfer learning tutorial written by @chsasank . It is a nice tutorial in general. However, I find some deviation from common practice:

- The fine-tuning dataset is just a subset of ImageNet.

- Only two classes for classification, making the problem trivial to solve.

- The de facto practice is to set the learning rate of new layers to be 10x of pre-trained layers, which is missing in the tutorial.

I plan to improve the transfer learning tutorial (more realistic datasets, more classes, comparison with training from scratch, etc), and also introduce a new co-tuning method (my NeurIPS'2020 paper for fine-tuning).

Since this would be a big change for the transfer learning tutorial, I'm putting a proposal here at first. After some positive feedback, I will proceed. 

What do you think?

**Comment by holly1238** [comment]
Updates in https://github.com/pytorch/tutorials/pull/1614
